src/plugins/Wallet/web3.ts

```diff
@@ -10,16 +10,21 @@ export const web3 = new Web3()
 let provider: WebsocketProvider
 
 export const resetProvider = () => {
-    if (provider) {
-        provider.removeListener('end', resetProvider) // prevent from circular reseting
-        provider.disconnect(1000, 'change provider')
-    }
-    console.log('Try to connect web3')
+    if (provider) provider.disconnect(1000, 'change provider')
```

Comment:
Is it a solution or you're trying to fix the problem?👀

Reply:
I found that there is a builtin reconnect solution was provided by `WebsocketProvider`. So I committed this patch. It might fix the syncing problem too.